include/klee/Config/config.h.cmin

```diff
@@ -86,9 +86,15 @@
 /* klee-uclibc is supported */
 #cmakedefine SUPPORT_KLEE_UCLIBC @SUPPORT_KLEE_UCLIBC@
 
+/* Path to klee-uclibc bca file */
```

Comment:
Why do added this one?
We currently have a single directory for the KLEE's runtime libraries that should be enough.

Reply:
See above. I agree with you that this seems like we could improve it, but this patch is from before #1056 was opened or merged, which is the one that added `KLEE_LIBCXX_BC_NAME` alongside `KLEE_UCLIBC_BCA_NAME`. I would suggest that we address this in an additional PR, if we want to.